persist/postgres/contracts_test.go

```diff
@@ -32,7 +33,11 @@ func TestFormRenewContract(t *testing.T) {
 		contract, err := store.Contract(context.Background(), id)
 		if err != nil {
 			t.Fatal("failed to fetch contract", err)
-		} else if !reflect.DeepEqual(contract, expected) {
+		} else if contract.Formation.Before(start) || contract.Formation.After(time.Now()) {
```

Comment:
```suggestion
		} else if contract.Formation.Before(start) || contract.Formation.After(time.Now().Round(time.Microsecond)) {
```

My two suggestions avoid an NDF